low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -85,13 +86,45 @@ def process(self):
                  "RABBITMQEGRESSPROCESSOR>password", rmq_passwd)
         Conf.save(SSPL_TEST_CONFIG_INDEX)
 
+        # TODO: Move lines 90-116 & 125-127 to RunQATest class
+        # Create dummy service and add service name in /etc/sspl.conf
+        service_name = "dummy_service.service"
+        service_file_path = f"{TEST_DIR}/alerts/os/dummy_service_files/dummy_service.service"
+        service_executable_code = f"{TEST_DIR}/alerts/os/dummy_service_files/dummy_service.py"
+        shutil.copy(service_executable_code, '/tmp/sspl/dummy_service.py')
+        # Make service file executable.
+        cmd = "chmod +x /tmp/sspl/dummy_service.py"
+        _, error, returncode = SimpleProcess(cmd).run()
+        if returncode !=0:
+            print("%s error occurred while executing cmd: %s" %(error, cmd))
+        # Copy service file to /etc/systemd/system/ path.
+        shutil.copyfile(service_file_path, '/etc/systemd/system/dummy_service.service')
+        cmd= "systemctl daemon-reload"
+        _, error, returncode = SimpleProcess(cmd).run()
+        if returncode !=0:
+                print("%s error occurred while executing cmd: %s" %(error, cmd))
+        cmd = "systemctl enable dummy_service.service"
+        _, error, returncode = SimpleProcess(cmd).run()
+        if returncode !=0:
+                print("%s error occurred while executing cmd: %s" %(error, cmd))
+        self.dbus_service.start('dummy_service.service')
+
+        service_list = Conf.get(SSPL_CONFIG_INDEX, "SERVICEMONITOR>monitored_services")
+        service_list.append(service_name)
+        Conf.set(SSPL_CONFIG_INDEX, "SERVICEMONITOR>monitored_services",
```

Comment:
`SSPL needs to read monitored services list in every iteration using confstor apis`
Do you also mean in ServiceMonitor??
if yes then we would have to change the logic to refer monitored_services list every iteration and remove disabled services in every iteration then remove already monitored_services from them and try to register remaining services.

Reply:
yes but, we are removing the disabled services only once and registered services as they get registered. Some level of refactoring in terms of lists will be needed since we are reading from confstor in order to compare any new service added or removed from confstor list.

Reply:
yes

Reply:
this you handle separately while addressing the need to avoid restarting service for tests

Reply:
sure. you mean in some next PR right??

Reply:
understood and noted.

Reply:
yes